pontoon/base/models.py

```diff
@@ -373,7 +375,8 @@ def locale_checkout_path(self, locale):
         if not self.multi_locale:
             raise ValueError('Cannot get locale_checkout_path for non-multi-locale repos.')
 
-        return os.path.join(self.checkout_path, locale.code)
+        path_components = [self.project.checkout_path] + urlparse(self.url).path.split('/')
+        return os.path.join(*path_components).format(locale_code=locale.code)
```

Comment:
I've changed it because i thought about urls which don't have locale in last part of url (e.g. '/{locale_code}/smth'. Anyway, i can restore this - no problem.